packages/components/pie-lottie-player/test/accessibility/pie-lottie-player.spec.ts

```diff
@@ -0,0 +1,18 @@
+
+import { test, expect } from '@justeattakeaway/pie-webc-testing/src/playwright/webc-fixtures.ts';
+import { PieLottiePlayer, LottiePlayerProps } from '../../src/index.ts';
+
+test.describe('PieLottiePlayer - Accessibility tests', () => {
+    test('a11y - should test the PieLottiePlayer component WCAG compliance', async ({ makeAxeBuilder, mount }) => {
+        await mount(
+            PieLottiePlayer,
+            {
+                props: {} as LottiePlayerProps,
+            },
+        );
```

Comment:
Can be simplified as we're not passing any props in
```suggestion
import { PieLottiePlayer } from '../../src/index.ts';

test.describe('PieLottiePlayer - Accessibility tests', () => {
    test('a11y - should test the PieLottiePlayer component WCAG compliance', async ({ makeAxeBuilder, mount }) => {
        await mount(PieLottiePlayer);
```